### DISCUSSION TITLE
Reduce the frequency of `queue/get-job-info?limit=50&dontExtendSession=1` calls

### DISCUSSION THREAD

**Discussion by boudewijn-zicht** [craftcms/cms#10838] [Ideas] [answered]
Hello Craft,

We have see that every day there are thousands of calls to `queue/get-job-info` from the CMS.  This is usually not an issue, but on our site (with many jobs and many users active in the CP) we see more than 150.000 calls to this url `https://cms.com/index.php?p=admin/actions/queue/get-job-info&limit=50&dontExtendSession=1` per day.

We would love to influence the frequency of these calls by influencing the hardcoded 500ms in `CP.js` in the below snippet (see https://github.com/craftcms/cms/blob/develop/src/web/assets/cp/src/js/CP.js#L1006)

```js
            // Determine the delay based on how long the displayed job info has remained unchanged
            var timeout = Math.min(60000, this.displayedJobInfoUnchanged * 500);
            this.trackJobProgressTimeout = setTimeout(this._trackJobProgressInternal.bind(this), timeout);
```

Could this possibly become something we can configure somewhere?

**Comment by brandonkelly** [comment, marked as answer]
We’re discussing how this could be improved over at #10632.
